MANIFEST.in

```diff
@@ -2,4 +2,4 @@ include *.rst
 include *.txt
 include tests/*.py
 include tox.ini
-include whitelists/*.py
+include vulture/whitelists/*.py
```

Comment:
Commenting here since __init__.py is an empty file: I don't think we need `whitelists/__init__.py`, please remove it.